packages/types/src/sdk/events/userGroup.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseEvent } from "./event"
+
+export interface GroupCreatedEvent extends BaseEvent {
+  groupId: string
+}
+
+export interface GroupUpdatedEvent extends BaseEvent {
+  groupId: string
+}
+
+export interface GroupDeletedEvent extends BaseEvent {
+  groupId: string
+}
+
+export interface GroupUsersAddedEvent extends BaseEvent {
+  emails: string[]
```

Comment:
Mentioned above, this should be removed. Could be replaced with `userIds`, or could just go away if we don't have any intention for tracking specific users in groups in analytics